Makefile

```diff
@@ -92,9 +92,9 @@ else
   include Makefile.docker
 
   # Fetch and install Bazel if needed
-  .PHONY: install-bazel
-  install-bazel:
-	tools/install_bazel.sh `cat .bazelversion`
+  .PHONY: install-bazelisk
+  install-bazelisk:
+	tools/bazelisk.sh
```

Comment:
Is `tools/baselisk.sh` already in the code base?

Reply:
good catch, thanks.